jormungandr/src/main.rs

```diff
@@ -592,6 +603,20 @@ fn initialize_node() -> Result<InitializedNode, start_up::Error> {
 
     let settings = raw_settings.try_into_settings()?;
 
+    let persistent_fragment_log = settings
```

Comment:
Is there a reason this is not done in the `start_services` function to avoid passing it through a bunch of other functions and sturcts?